javascriptv3/example_code/cross-services/ddb-item-tracker/README.md

```diff
@@ -0,0 +1,729 @@
+            const data = await s3Client.send(new PutObjectCommand(uploadParams));
+            const linkToCSV =
+                "https://s3.console.aws.amazon.com/s3/buckets/" +
+                uploadParams.Bucket +
+                "?region=" +
+                REGION +
+                "&tab=objects"
+            console.log("Success. Report uploaded to " + linkToCSV + ".");
+            console.log('Email', req.body.email)
+            try{
+                const emailParams = {
+                    Destination: {
+                        /* required */
+                        CcAddresses: [
+                            /* more items */
+                        ],
+                        ToAddresses: [
+                            req.body.email, //RECEIVER_ADDRESS
+                            /* more To-email addresses */
+                        ],
+                    },
+                    Message: {
+                        /* required */
+                        Body: {
+                            /* required */
+                            Html: {
+                                Charset: "UTF-8",
+                                Data:
+                                    "<h1>Hello!</h1>" +
+                                    "<p> The CSV has been uploaded to <a href=" +linkToCSV +
+                                    ">here.</a></p>"
+                            },
+                        },
+                        Subject: {
+                            Charset: "UTF-8",
+                            Data: "Report ready.",
+                        },
+                    },
+                    Source: "brmur@amazon.com",
+                    ReplyToAddresses: [
+                        /* more items */
+                    ],
+                };
+                const data = await sesClient.send(new SendEmailCommand(emailParams));
+                console.log("Success. Email sent.", data);
+                res.contentType = "application/json";
+                res.send(data);
+            } catch (err) {
+                console.log("Error", err);
+            };
+        } catch (err) {
+            console.log("Error", err);
+        }
+    }
+    run();
+});
+
+
+```
+The final section binds and listens the connections on the specified host and port, which in this case is 3000, 
+as defined at the top of **app.js**.
+```javascript
+app.listen(port, () => {
+    console.log(`Listening on port ${port}`);
+})
+```
+
+### <a name="destroy"></a>Destroy the resources
+
+If you created your resources through the Amazon Management Console, you must delete all your resource manually through Console.
+
+However, if you created your resources using Amazon CloudFormation, you can delete them by deleting the stack.
+**Note:** By running the app you modified the table, and the bucket, so before you can delete the stack,  you must delete these resources manually via the Amazon Management Console before deleting the Stack.
```

Comment:
Need another line break to put the note on a new line.

Reply:
done